### ISSUE TITLE
Not able to build it

### ISSUE THREAD

**Issue by michalss** [2sic/oqtane-theme-2shine-bs5#4]
Hi i did try it, buti was not able to build it... i followed steps from from here https://github.com/2sic/oqtane-theme-2shine-bs5/blob/main/docs/gettings-started.md

Logs:
[_errors.txt](https://github.com/2sic/oqtane-theme-2shine-bs5/files/10611579/_errors.txt)

**Comment by michalss** [comment]
ok i had to clone https://github.com/2sic/cre8magic and change the path in project... but this is not included in steps..

**Comment by michalss** [comment]
made it work 

![image](https://user-images.githubusercontent.com/1000729/216833582-bfab5549-a071-41a9-9112-565707a2297b.png)


But how can i do this very nice looking theme from your example you did show ?

**Comment by michalss** [comment]
But still, i dont have any idea how to work with this... :(

**Comment by michalss** [comment]
also this 
![image](https://user-images.githubusercontent.com/1000729/216833802-c9730733-d102-433e-bfd6-ae24017abbf0.png)

**Comment by michalss** [comment]
alos missing 2sic container..
![image](https://user-images.githubusercontent.com/1000729/216833921-88e43398-b8fd-4f80-bb7a-cd95f2a6af34.png)

**Comment by iqbal-skd** [comment]
Any plans to update the code for .net 9 and oqtane 6.0.1? also when I upgrade the .net version myself to .net it works and installs the theme but when I try to apply the theme, it does not load any container which causes an error.
<img width="688" alt="Screenshot 2024-12-28 050734" src="https://github.com/user-attachments/assets/ddaebd08-fcdc-4844-82a1-87458efced7a" />

**Comment by iJungleboy** [comment]
@iqbal-skd yes, we've been actively developing a brand-new cre8magic for the last 4 months or so. That's the foundation of the next 2shine theme. So we hope to have something within 6-8 weeks from today.
